### PR TITLE
Fixes #7306

### LINKED ISSUE #7306: The serialization of porphyrin to mol format introduces some double bonds with bond stereo 3/either
**Describe the bug**
Loading a structure containing a porphyrin into an RDKit molecule instance and rewriting it to mol format introduces some  double bonds with bond stereo 3/either

**To Reproduce**

    In [1]: from rdkit import Chem
    
    In [2]: print(Chem.MolToMolBlock(Chem.MolFromMolFile('./input.mol')))

[input.zip](https://github.com/rdkit/rdkit/files/14775313/input.zip)

**Configuration (please complete the following information):**
 - RDKit version: current tip of the master branch (I presume the issue may also affect the 2024.03.1 beta)
 - OS: Fedora 39
 - If you are not using conda: how did you install the RDKit? from source code

### PR DESCRIPTION
**Pull request by greglandrum**
Do not cross bonds which were previously aromatic

### CODE REVIEW

**Comment by greglandrum** [comment]
@rvianello here's a proposed fix for this issue

**Comment by rvianello** [comment]
@greglandrum thank you for this fix, it worked as desired on a couple of structures I tested. Thank you also for the explanation posted to #7306.

File: Code/GraphMol/ChemReactions/ReactionWriter.cpp [hunk 1]
```diff
@@ -113,7 +113,8 @@ void write_template(std::ostringstream &res, RDKit::ROMol &tpl) {
   }
   RDKit::FileParserUtils::moveAdditionalPropertiesToSGroups(trwmol);
 
-  res << RDKit::FileParserUtils::getV3000CTAB(trwmol, -1);
+  boost::dynamic_bitset<> wasAromatic(trwmol.getNumBonds());
+  res << RDKit::FileParserUtils::getV3000CTAB(trwmol, wasAromatic, -1);
```
Comment (bp-kelley) [inline Code/GraphMol/ChemReactions/ReactionWriter.cpp:117]: Due to the new getV3000CTab internally using the wasAromatic bitset, does this need to be changed or can it be the original:

```
res << RDKit::FileParserUtils::getV3000CTAB(trwmol, -1);
```

Comment (greglandrum) [reply Code/GraphMol/ChemReactions/ReactionWriter.cpp:117]: you're right; there's no reason to change this.

**Comment by greglandrum** [comment]
@bp-kelley I made the change you suggested

**Pull request merged**